src/views/SingleCommentView.vue

```diff
@@ -0,0 +1,184 @@
+              placeholder="Add reply"
+              class="form-control reply__input"
+            >
+          </div>
+        </div>
+        <div class="send-comment">
+          <ae-button
+            :disabled="newComment.length === 0 || showLoading"
+            @click="sendTipComment()"
+          >
+            Reply
+          </ae-button>
+        </div>
+      </div>
+      <div class="comments__section">
+        <div
+          v-if="comments.length === 0 && !showLoading"
+          class="no-results text-center w-100"
+          :class="[error ? 'error' : '']"
+        >
+          {{ $t('pages.TipComments.NoResultsMsg') }}
+        </div>
+
+        <!-- <tip-comment
+          v-for="(comment, index) in comments"
+          :key="index"
+          :comment="comment"
+        /> -->
+        <div
+          v-if="showLoading"
+          class="text-center w-100 mt-3"
+        >
+          <loading :show-loading="true" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+// eslint-disable-next-line import/no-cycle
+import Backend from '../utils/backend';
+// import { USE_DEEP_LINKS } from '../utils/util';
+import TipComment from '../components/tipRecords/TipComment.vue';
+import LeftSection from '../components/layout/LeftSection.vue';
+import RightSection from '../components/layout/RightSection.vue';
+// eslint-disable-next-line import/no-cycle
+import MobileNavigation from '../components/layout/MobileNavigation.vue';
+// import { wallet } from '../utils/walletSearch';
+import Loading from '../components/Loading.vue';
+import { EventBus } from '../utils/eventBus';
+import AeButton from '../components/AeButton.vue';
+import Avatar from '../components/Avatar.vue';
+
+export default {
+  name: 'CommentView',
+  components: {
+    Loading,
+    TipComment,
+    LeftSection,
+    RightSection,
+    MobileNavigation,
+    AeButton,
+    Avatar,
+  },
+  data() {
+    return {
+      id: this.$route.params.id,
+      showLoading: true,
+      comments: [],
+      error: false,
+      newComment: '',
+      address: null,
+      comment: null,
+      tipId: this.$route.params.tipId,
+    };
+  },
+  computed: {
+    ...mapGetters(['settings', 'account', 'chainNames', 'isLoggedIn']),
+  },
+  // watch: {
+  //   tip() {
+  //     this.updateTip();
+  //   },
+  // },
+  created() {
+    this.loadComment();
+    const loadUserAvatar = setInterval(() => {
+      if (this.isLoggedIn) {
+        this.address = this.account;
+        clearInterval(loadUserAvatar);
+      }
+    }, 500);
+
+    EventBus.$on('reloadData', () => {
+      this.reloadData();
+    });
+
+    this.interval = setInterval(() => this.reloadData(), 120 * 1000);
+  },
+  beforeDestroy() {
+    clearInterval(this.interval);
+  },
+  methods: {
+    getAvatar(address) {
+      const userImage = Backend.getProfileImageUrl(address);
+      return userImage || this.avatar;
+    },
+    // async sendTipComment() {
```

Comment:
can you clean up these, or it is still WIP

Reply:
This should be implemented - sending comment to another comment